websites/P/PreMiD/presence.ts

```diff
@@ -57,13 +58,23 @@ presence.on("UpdateData", async () => {
 			largeImageKey: "lg"
 		},
 		newLang = await presence.getSetting<string>("lang"),
-		time = await presence.getSetting<boolean>("time");
+		time = await presence.getSetting<boolean>("time"),
+		showButtons = await presence.getSetting<boolean>("showButtons");
 
 	if (oldLang !== newLang || !strings) {
 		oldLang = newLang;
 		strings = await getStrings();
 	}
 
+	if (showButtons) {
+		presenceData.buttons = [
+			{
+				label: (await strings).btnViewPage,
```

Comment:
You don't need to `await` if you have the Awaited type

Reply:
So should I replace all `(await strings)` or just this one?